lib/vkloader.c

```diff
@@ -1763,18 +1873,62 @@ generateMipmaps(ktxVulkanTexture* vkTexture, ktxVulkanDeviceInfo* vdi,
  * @brief Destructor for the object returned when loading a texture image.
  *
  * Frees the Vulkan resources created when the texture image was loaded.
+ * If a complete set of suballocator callbacks
+ * are provided, it will use the free call from those instead.
  *
- * @param vkTexture  pointer to the ktxVulkanTexture to be destructed.
- * @param device     handle to the Vulkan logical device to which the texture was
- *                   loaded.
- * @param pAllocator pointer to the allocator used during loading.
+ * @param vkTexture                pointer to the ktxVulkanTexture to be destructed.
+ * @param device                   handle to the Vulkan logical device to which the texture was
+ *                                 loaded.
+ * @param pAllocator               pointer to the allocator used during loading.
+ * @param subAllocatorCallbacks    potential function pointer for guarded suballocator callbacks.
```

Comment:
Change description to

"pointer to a structure of guarded suballocator function callbacks."

Reply:
Done! 👍